### PR TITLE
TE5-T238 Basics of kotlin

### PR DESCRIPTION
**Pull request by jay-a-shah**

### CODE REVIEW

File: app/src/main/java/com/example/demoappkotlin/BasicsKotlin.kt [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+import androidx.appcompat.app.AppCompatActivity
+import android.os.Bundle
+import android.util.Log
+
+class BasicsKotlin : AppCompatActivity() {
+    override fun onCreate(savedInstanceState: Bundle?) {
+        super.onCreate(savedInstanceState)
+        setContentView(R.layout.activity_basics_kotlin)
+
+
```
Comment (purvesh-d) [inline app/src/main/java/com/example/demoappkotlin/BasicsKotlin.kt:12]: Remove this extra spaces.

File: app/src/main/java/com/example/demoappkotlin/MainActivity.kt [hunk 2]
```diff
@@ -2,10 +2,93 @@ package com.example.demoappkotlin
...
+        Log.d("Functions",z.toString())
+
+        //Classes
+        class Rectangle(var height:Double,var Breadth:Double){
+            var perimeter = (height+Breadth*2).toString()
+        }
+        var rectangle = Rectangle(6.0,2.0)
+        Log.d("Class",rectangle.perimeter)
+//Operators
+        var p1 = 13
```
Comment (purvesh-d) [inline app/src/main/java/com/example/demoappkotlin/MainActivity.kt:29]: Apply code formatting in All files.

**Review by purvesh-d — CHANGES_REQUESTED** [review]

File: app/src/main/java/com/example/demoappkotlin/BasicsKotlin.kt (hunk 1 above)
Comment (jay-a-shah) [reply app/src/main/java/com/example/demoappkotlin/BasicsKotlin.kt:12]: Okk I will resolve it

File: app/src/main/java/com/example/demoappkotlin/MainActivity.kt (hunk 2 above)
Comment (jay-a-shah) [reply app/src/main/java/com/example/demoappkotlin/MainActivity.kt:29]: Okk

**Review by purvesh-d — CHANGES_REQUESTED** [review]
Add gitignore file and .idea and autogenerated files should not included in PR.

File: .idea/gradle.xml [hunk 3]
```diff
@@ -1,5 +1,6 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <project version="4">
+  <component name="GradleMigrationSettings" migrationVersion="1" />
```
Comment (purvesh-d) [inline .idea/gradle.xml]: Add this file in gitignore

File: app/src/main/java/com/example/demoappkotlin/MainActivity.kt [hunk 4]
```diff
@@ -2,10 +2,237 @@ package com.example.demoappkotlin
...
+
+        //Classes
+
+        class Rectangle(var height: Double, var Breadth: Double) {
+            var perimeter = (height + Breadth * 2).toString()
+        }
+
+        var rectangle = Rectangle(6.0, 2.0)
+        Log.d("Class", rectangle.perimeter)
+
```
Comment (purvesh-d) [inline app/src/main/java/com/example/demoappkotlin/MainActivity.kt]: Extra empty lines

**Review by purvesh-d — CHANGES_REQUESTED** [review]

File: app/src/main/java/com/example/demoappkotlin/MainActivity.kt (hunk 4 above)
Comment (jay-a-shah) [reply app/src/main/java/com/example/demoappkotlin/MainActivity.kt]: ok

File: .idea/gradle.xml (hunk 3 above)
Comment (jay-a-shah) [reply .idea/gradle.xml]: ok

**Pull request merged**